openlibrary/templates/admin/people/view.html

```diff
@@ -112,7 +112,7 @@ <h1><a class="plain" href="/admin/people">$_("People")</a> / $person.username</h
         $if person.status in ['active', 'verified']:
             <button class="do-confirm" type="submit" name="action" value="block_account_and_revert" style="float: right; background: #ffbbbb;">$_("block and revert all edits")</button>
             <button class="do-confirm" type="submit" name="action" value="block_account">$_("block this account")</button>
-            &nbsp;&nbsp;<button type="submit" name="action" value="send_password_reset_email">$_("send password reset email")</button>
+            <!-- &nbsp;&nbsp;<button type="submit" name="action" value="send_password_reset_email">$_("send password reset email")</button> -->
```

Comment:
```suggestion
```

Reply:
Let's delete this instead of commenting it out. With this done, I think we can merge this PR.